--- jnius_utils.pxi.original	2025-08-09 12:00:00.000000000 +0000
+++ jnius_utils.pxi.patched	2025-08-09 12:00:00.000000000 +0000
@@ -320,7 +320,7 @@
             score += 10
             continue
 
-        if r == 'S' or r == 'I':
-            if isinstance(arg, int) or (
-                    (isinstance(arg, long) and arg < 2147483648)): 
+        if r == 'S' or r == 'I':
+            if isinstance(arg, int) and arg < 2147483648:
                 score += 10
                 continue